### ISSUE TITLE
questionable options to a11y mixins

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#388]
In AccessibleSlider, anything in the default options (`defaults`, line 79) can be changed by the client.  Do you really want the client to be able to change `ariaRole`, `tagName`,… other things?  As currently implemented, I could (for example) pass in `ariaRole: 'spinbutton'` and there would be no complaint.

Same question about AccessibleNumberSpinner.

If there are options that should not be set by the client, they should not be in defaults. They should be explicitly added to options after asserting that they have not been provided by the client.

**Comment by jessegreenberg** [comment]
Thanks @pixelzoom, `tagName`, `inputType`, and `ariaRole` have been removed from AccessibleSlider defaults so they can't be set by the client. Same for AccessibleNumberSpinner. Would you mind reviewing the above commits?
